_src/_assets/js/d3-area-chart.js

```diff
@@ -103,6 +103,13 @@ function d3AreaChart({
     .axisLeft()
     .scale(yScale)
     .ticks(4)
+    .tickFormat(function(d){
+      if(d/1000000 < 1){
+        return d/1000000;
+      } else {
+        return d/1000000 + " Million";
```

Comment:
This is a fine, approach but I actually think we just want this to happen on the first tick and not any tick over a million. Also I think the "million" should be lowercase.